gatsby-theme-landing-page/src/components/features.js

```diff
@@ -1,10 +1,55 @@
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
 import * as React from "react";
 import * as styles from "./features.module.css";
+import MarkdownText from "./markdown-text";
+import Link, { LinkContainer } from "./link";
```

Comment:
Not blocking for this PR, but I'd recommend keeping shared components 1:1 with files for finer granularity with shadowing